.prettierignore

```diff
@@ -19,6 +19,7 @@ build/
 /files/fr/glossary/grid_axis/index.md
 /files/fr/glossary/grid_lines/index.md
 /files/fr/glossary/main_axis/index.md
+/files/zh-cn/learn/css/howto/css_faq/index.md
```

Comment:
Seems that this file is also formatted 🤔

Reply:
It is, but there's one change that Prettier produces with a link in that file that breaks the link...  Seems to be common with this specific file.  :/

Reply:
Let me open another PR to deal with this file :(